schema/logs/communication.mapping

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
Do schema and mapping convey the same information. I understood the purpose of mapping where users can easily create mappings of indices using these templates. What is the purpose of schema files?

Reply:
schema file will help builders enforce structure and content since mapping file doesn’t have such capabilities or notions 
many json enforcement tools exist and can help any developer validate and understand the schema we support